src/glb.rs

```diff
@@ -6,47 +6,118 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+use byteorder::{LE, ReadBytesExt};
 
 use Error;
 
+use std::{fmt, io};
+
+/// Represents a Glb loader error.
+#[derive(Debug)]
+pub enum GlbError {
```

Comment:
I have an aversion toward repeated prefixes and suffixes. `Error` is just fine, as the crate will use `glb::Error` and the user `gltf::glb::Error`.